proto/decentraland/sdk/components/mesh_collider.proto

```diff
@@ -6,25 +6,39 @@ import "decentraland/sdk/components/common/id.proto";
 
 option (common.ecs_component_id) = 1019;
 
+// The MeshCollider component provides basic collision detection between Entities and Avatars. It 
+// can behave like a cube, a plane, a sphere or a cylinder. 
+
+// The Entity's Transform affects the MeshCollider, scaling it and rotating it appropriately. If
+// the Transform for the collision mesh must be different than the Transform for rendering the Entity,
+// two different Entities must be used.
+
+// More complex shapes require the use of a `GltfContainer` component.
+
+// ColliderLayer determines the kind of collision to detect, in OR-able bit flag form.
 enum ColliderLayer {
-  CL_NONE = 0;
-  CL_POINTER = 1;
-  CL_PHYSICS = 2;
+  CL_NONE = 0;    // no collisions
+  CL_POINTER = 1; // collisions with the player's pointer ray (e.g. mouse cursor hovering)
+  CL_PHYSICS = 2; // colission with Avatars
```

Comment:
great question
IDK if it's already included in CL_POINTER, worth checking, but I prefer the approach of it living as another option in the enum

Reply:
Shall we add it? We can use the CL_ as combinable flags. Would you bring it to the daily tomorrow?

Reply:
From conversations we just had, we concluded that it would be ideal to configure these things from the raycast itself. 

So instead of creating a new separate raycast collision layer, we should add a property to the raycast, to select which layers to use, to use the logic that makes the most sense for that particular ray.

Reply:
In my humble opinion, that seems like a much more flexible and future-proof approach.